app/mailers/notifier.rb

```diff
@@ -75,7 +75,7 @@ def attach_statement_pdf
   end
 
   def statement_pdf
-    @statement_pdf ||= StatementPdfFactory.instance(@statement)
+    @statement_pdf ||= ::StatementPdfFactory.instance(@statement)
```

Comment:
Did you test calling `StatementPdfFactory` without `::` with the defining file in `app/lib`?